pkgs/development/ocaml-modules/notty/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, buildOcaml, fetchFromGitHub, findlib, result, uucp, uuseg, uutf, lwt ? null}:
+
+buildOcaml rec {
+  version = "0.1.1";
+  name = "notty";
+
+  minimumSupportedOcamlVersion = "4.02";
+
+  src = fetchFromGitHub {
+    owner  = "pqwy";
+    repo   = "notty";
+    rev    = "v${version}";
+    sha256 = "0bw3bq8z2y1rhc20zn13s78sazywyzpg8nmyjch33p7ypxfglf01";
+  };
+
+  buildInputs = [ findlib ];
+  propagatedBuildInputs = [ lwt result uucp uuseg uutf ];
```

Comment:
Yeah, indeed. If a package's interface is used in the module, it needs the package propagated, I guess.